### PR TITLE
Adding support for external authentication / access token generation

### PR DESCRIPTION
**Pull request by kevinterrobang**
I am using passport (node authentication middleware) for my user authentication with an additional Instagram authentication module. As this means I'm managing the authentication and access token externally, I wanted a way to initialize Instajam with an existing token and just use the functional endpoints.

### CODE REVIEW

File: dist/instajam.js [hunk 1]
```diff
@@ -48,6 +48,21 @@
     // constructor parameters.
     this.authUrl = 'https://instagram.com/oauth/authorize/?client_id=' + options.clientId + '&redirect_uri=' + options.redirectUri + '&response_type=token' + (this.scope || '');
 
+    // If an access token has already been generated, setup that one
+    // instead of requesting our own
+    if (options.accessToken) {
+
```
Comment (mikefowler) [inline dist/instajam.js]: Could you just get rid of this extra newline?

**Comment by mikefowler** [comment]
Seems legit, @kevinterrobang. If you could address the one comment above I'll merge this.

**Comment by kevinterrobang** [comment]
How did that get there?! Removed the stray newline from the source and from the non-minified distribution file.

**Comment by mikefowler** [comment]
Thanks!

**Pull request merged**